apps/communications/dialer/js/oncall.js

```diff
@@ -8,6 +7,9 @@
 function onLoadCallScreen(evt) {
   window.removeEventListener('load', onLoadCallScreen);
 
+  // According to the UX sound spec, we should use the "system" type for the
+  // dialer pad. See the attachment of the bug1068219.
+  TonePlayer.init('telephony');
```

Comment:
Yes, this is used only during a call. If I use the system type to play the end of call sound for example it plays through the loudspeaker instead of the phone/headset. This is used to play all sounds during a call (key presses, busy tone, end-of-call tone, etc...).